pkg/packagekit/autoupdates.jsx

```diff
@@ -320,41 +380,53 @@ export default class AutoUpdates extends React.Component {
-                </div>
-            );
-        }
-
-        // we want the button to already show the target state while being disabled
-        const onOffState = this.state.pendingEnable == null ? backend.enabled : this.state.pendingEnable;
-
-        return (
-            <div className="header-buttons pk-updates--header pk-updates--header--auto" id="automatic">
-                <h2 className="pk-updates--header--heading">{_("Automatic updates")}</h2>
-                <div className="pk-updates--header--actions">
-                    <OnOffSwitch state={onOffState} disabled={!enabled}
-                                 onChange={e => {
-                                     if (!this.state.backend.installed) {
-                                         install_dialog(this.state.backend.packageName)
-                                                 .then(() => { this.initializeBackend(true).then(() => { this.handleChange(e, null, null, null) }) }, () => null);
-                                     } else {
-                                         this.handleChange(e, null, null, null);
-                                     }
-                                 }} />
-                </div>
-                {autoConfig}
-            </div>);
+                    </FormGroup>
+
+                    <Alert variant="info" title={_("This host will reboot after updates.")} isInline />
+                </>}
+            </Form>
+        );
+
+        return (<>
+            <Button variant="primary"
+                    onClick={() => {
+                        if (!this.state.backend) {
+                            install_dialog(this.state.backend.packageName)
+                                    .then(() => {
+                                        this.initializeBackend(true);
+                                        this.setState({ showModal: true });
+                                    }, () => null);
+                        } else {
+                            this.setState({ showModal: true });
+                        }
+                    }}>
+                {_("Edit")}
+            </Button>
+            <Modal position="top" variant="medium" id="automatic-updates-dialog" isOpen={this.state.showModal} onClose={() => this.setState({ showModal: false })}
+                   title={_("Automatic updates")}
+                   footer={
+                       <>
+                           <Button variant="primary"
+                                   isLoading={ this.state.pending }
+                                   isDisabled={ this.state.pending }
+                                   onClick={ this.handleChange }>
+                               {_("Save changes")}
+                           </Button>
+                           <Button variant="link" className="btn-cancel" onClick={() => this.setState({ showModal: false })}>
```

Comment:
btn-cancel class not used anywhere. You can just drop it

Reply:
Fixed